lib/cadet_web/views/assessments_view.ex

```diff
@@ -123,6 +123,7 @@ defmodule CadetWeb.AssessmentsView do
     transform_map_for_view(answer, %{
       answer: answer_getter,
       comment: :comment,
+      gradedBy: & &1.grader[:name],
```

Comment:
You need to make sure to preload `grader` in `Assessments.assessment_with_questions_and_answers/2`

Reply:
already being preloaded.